README.md

```diff
@@ -65,15 +65,17 @@ Crescent core uses a fork of [cosmos-sdk](https://github.com/crescent-network/co
 
 | Requirement         | Notes                |
 |---------------------|----------------------|
-| cosmos-sdk (forked) | v1.2.0-sdk-0.45.10   |
-| ibc-go (forked)     | v3.4.0-crescent-v4-2 |
+| cosmos-sdk (forked) | v1.1.5-sdk-0.45.10   |
```

Comment:
Don't we need to update `v1.2.0` to `v1.2.1` if the patch is need?

Reply:
`1.2.0` was released for `MinDepositRatio` but, we decided to implement this feature on antehandler level https://github.com/crescent-network/crescent/pull/159